xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.cpp

```diff
@@ -215,6 +215,24 @@ int CDVDInputStreamPVRManager::GetTime()
   return 0;
 }
 
+bool CDVDInputStreamPVRManager::GetTimes(Times &times)
+{
+  if (m_isRecording)
+    return false;
+
+  PVR_STREAM_TIMES streamTimes;
+  bool ret = CServiceBroker::GetPVRManager().Clients()->GetStreamTimes(streamTimes);
+  if (ret)
```

Comment:
This would be the place for implementing migration - old functions vs. new getstreamtimes functions, right?